integration/examples/kaniko/skaffold.yaml

```diff
@@ -6,7 +6,8 @@ build:
       kaniko:
         cache: {}
   cluster:
-    pullSecretName: e2esecret
+    pullSecretName: e2esecret-with-path
+    pullSecret:     kaniko-secret.json
```

Comment:
this is not going to work unless we recreate this secret for each integration test, right? I think kokoro should fail (also because of the other bug I commented on)

Reply:
we don't need to recreate the secret.  
This test make sure, if an existing secret with file other than `kaniko` is loaded using the `kubectl create` command, users should be able to use this secret correctly.
by specifying the right config.
e.g. secret created using, following command.

```
 kubectl create secret generic kaniko-secret --from-file=/Users/tejaldesai/workspace/keys/my-key.json`
```

```
cluster:
    pullSecretName: e2esecret-with-path
    pullSecret:   my-keys.json
```